### PR TITLE
[no-issue] Bumped versionig matrix versions to 1020

### PR DESCRIPTION
**Pull request by eniosultan**
## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
<br/>

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/SoftwareAG/cumulocity-community-plugins/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/SoftwareAG/cumulocity-community-plugins/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by reey** [comment]
So we never had a version that was supported on 1018?

File: versioningMatrix.json [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-      "sdk": ">=1018.0.0",
-      "api": ">=1018.0.0"
-    },
-    "3.2.2": {
-      "sdk": ">=1018.0.0",
-      "api": ">=1018.0.0"
-    }
+  "2.1.6": {
+    "sdk": ">=1016.0.0 <1018.0.0",
+    "api": ">=1016.0.0 <1018.0.0"
```
Comment (janhommes) [inline versioningMatrix.json:4]: So 1018 is never supported?

Comment (eniosultan) [reply versioningMatrix.json:4]: 3.0.0 is the version in which we bumped @c8y components from 1016 to 1020, so all versions after are indeed not compatible. We have a few versions before 2.1.6 which are not listed in the matrix, but I assume there was a reason for this...

**Comment by eniosultan** [comment]
> So we never had a version that was supported on 1018?

@reey  Hmm, from 3.0.0+ it is not compatible after we bumped ngx-components from 1016 to 1020, but for 2.1.6 I'm not sure why it is marked, I'll try it out

**Comment by eniosultan** [comment]
@janhommes  @reey  2.1.6 works fine for 1018 so I've adjusted the matrix.

**Review by janhommes — APPROVED** [review]

**Review by reey — APPROVED** [review]

**Pull request merged**